Client/Frontend/Strings.swift

```diff
@@ -249,11 +249,16 @@ extension String {
                 value: "Your search for \"%@\"",
                 comment: "On the Firefox homepage in the Jump Back In section, if a Tab group item - a collection of grouped tabs from a related search - exists, the Tab Group item title will be 'Your search for \"video games\"'. The %@ sign is a placeholder for the actual search the user did.",
                 lastUpdated: .v39)
-            public static let TabPickup = MZLocalizedString(
+            public static let SyncedTabTitle = MZLocalizedString(
                 "FirefoxHomepage.JumpBackIn.TabPickup.v104",
                 value: "Tab pickup",
                 comment: "If a user is signed in, and a sync has been performed to collect their recent tabs from other signed in devices, their most recent tab from another device can now appear in the Jump Back In section. This label specifically points out which cell inside the Jump Back In section shows that synced tab.",
                 lastUpdated: .v104)
+            public static let SyncedTabShowAllButtonTitle = MZLocalizedString(
+                "FirefoxHomepage.JumpBackIn.TabPickup.ShowAll.ButtonTitle.v104",
+                value: "See all synced tabs",
+                comment: "Button title shown for tab pickup on the Firefox homepage in the Jump Back In section.",
+                lastUpdated: .v104)
```

Comment:
Since the export already happened for .v104, this will be exported in .v105 - Just highlighting this here in case this would be a problem.

Reply:
The feature will not be release for a while and is behind a feature flag so this shouldn't be an issue.